### PR TITLE
[terra-dev-site] CVE fixes

### PR DESCRIPTION
**Pull request by sugan2416**
### Summary

- Fixes `terra-dev-site` CVEs 
- Removed `link-parent-bin` as deprecated package.

Closes #

### Deployment Link

https://terra-applic-.herokuapp.com/

### Testing

Updated WDIO screenshots and Jest snapshots due to package updates

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by JessieRandle** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please include a CHANGELOG entry for each changed package on this PR. Looks like a CHANGELOG is missing for: 

 - terra-dev-site</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5d64cd081a5ed9f9172c0d208d7b0ec60f6bb591
</p>

File: package.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
     "intl": "^1.2.5",
     "jest": "^26.6.3",
     "ky": "^0.12.0",
-    "lerna": "^3.16.4",
+    "lerna": "^6.4.1",
     "link-parent-bin": "^1.0.0",
```
Comment (supreethmr) [inline package.json]: remove link-parent-bin

Comment (sugan2416) [reply package.json]: Removed https://github.com/cerner/terra-application/pull/320/commits/6727052e7e30ed056d069c432949024171d3507a

File: packages/terra-application/src/terra-dev-site/app/components.2/ApplicationBase.app.mdx [hunk 2]
```diff
@@ -30,7 +30,7 @@ ApplicationBase exposes props to define the application's theme.
 
 ApplicationBase renders a theme context provider around its children. Children can access the current theme by using the [ThemeContext](/application/terra-application/contexts/theme-context).
 
-<!-- Consumers of ApplicationBase should review Terra's [theme aggregation](https://github.com/cerner/terra-toolkit-boneyard/blob/main/scripts/aggregate-themes/README.md) instructions to ensure the all theme properties are loaded appropriately. -->
+!-- Consumers of ApplicationBase should review Terra's [theme aggregation](https://github.com/cerner/terra-toolkit-boneyard/blob/main/scripts/aggregate-themes/README.md) instructions to ensure the all theme properties are loaded appropriately. --
```
Comment (saket2403) [inline packages/terra-application/src/terra-dev-site/app/components.2/ApplicationBase.app.mdx]:
```suggestion
Consumers of ApplicationBase should review Terra's [theme aggregation](https://github.com/cerner/terra-toolkit-boneyard/blob/main/scripts/aggregate-themes/README.md) instructions to ensure the all theme properties are loaded appropriately.
```

Comment (sugan2416) [reply packages/terra-application/src/terra-dev-site/app/components.2/ApplicationBase.app.mdx]: https://github.com/cerner/terra-application/pull/320/commits/ad798c27f4ac33ca4bced4f44443b07aa11d0cc4

**Review by supreethmr — APPROVED** [review]

**Review by supreethmr — CHANGES_REQUESTED** [review]
We have updated the props table screenshot for incorrect style value in this PR : https://github.com/cerner/terra-application/pull/319 
As you can see the 1st column ( prop names ) are expected to be displayed as bold but due to html-table changes https://github.com/cerner/terra-core/pull/3695/files#diff-1914341c0a4e12e578a435150e3253cafe5e6be59094713364fbaf116569f5e1 
[props table styles](https://github.com/cerner/terra-application/blob/02a2d1d55e19b604df8e19a36d3eae67f4417adc/packages/terra-dev-site/src/loader-components/PropsTable.module.scss#L6) were over-ridden

you need to re-generate screenshots for props-table by marking [bold style as important](https://github.com/cerner/terra-application/blob/40092103a2d936246f0eca5078ba16d7f89efc71/packages/terra-dev-site/src/loader-components/PropsTable.module.scss#L6)

**Review by supreethmr — APPROVED** [review]

**Comment by sugan2416** [comment]
Tested with Terra Core and Framework. Requires mdx syntax corrections.

**Pull request merged**